examples/kyb-app/src/pages/CollectionFlow/components/organisms/KYBView/helpers/transform-errors.ts

```diff
@@ -3,6 +3,7 @@ import { RJSFValidationError } from '@rjsf/utils';
 
 export const transformRJSFErrors = (errors: RJSFValidationError[]): RJSFValidationError[] => {
   return errors.map(error => {
+    console.log('error', error);
```

Comment:
Can this log be removed or be handled differently? Perhaps dev only or using `error.message` etc.

Reply:
This is leftover.